modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/DataSet.js

```diff
@@ -61,6 +61,9 @@ import {getViewContentRenderer} from './views/index';
 const DEFAULT_PAGINATION_DELTA = 20;
 const DEFAULT_PAGINATION_PAGE_NUMBER = 1;
 
+const getSelectedItems = ({items, key, values}) =>
+	items?.filter((item) => values.includes(item[key])) || [];
+
```

Comment:
We are setting values here from constantly changing `items`. This will overwrite selected items:

![screenshot](https://user-images.githubusercontent.com/5435511/165935780-7cd7c08d-3b02-459a-9523-39acdf62ce1a.gif)

Reply:
:(
I'm sorry, this should not happen; I'll find the way.

Reply:
Pushed! The mechanism has to be more complex that I have expected